src/main/java/org/folio/consortia/service/impl/TenantServiceImpl.java

```diff
@@ -95,12 +97,27 @@ public Tenant save(UUID consortiumId, UUID adminUserId, Tenant tenantDto) {
 
     try (var context = new FolioExecutionContextSetter(prepareContextForTenant(tenantDto.getId(), folioModuleMetadata, currentTenantContext))) {
       createShadowAdminUserWithPermissions(shadowAdminUser);
+      saveDummyUser(tenantDto.getId());
       configurationClient.saveConfiguration(createConsortiaConfigurationBody(centralTenantId));
     }
     log.info("save:: saved consortia configuration with centralTenantId={} by tenantId={} context", centralTenantId, tenantDto.getId());
     return savedTenant;
   }
 
+  private void saveDummyUser(String tenantId) {
+    UserTenant userTenant = createUserTenantWithDummyUser(tenantId);
+    userService.createUserTenant(userTenant);
+  }
+
+  private UserTenant createUserTenantWithDummyUser(String tenantId) {
+    UserTenant userTenant = new UserTenant();
+    userTenant.setId(UUID.randomUUID());
+    userTenant.setTenantId(tenantId);
+    userTenant.setUserId(UUID.randomUUID());
```

Comment:
Lets also extract id and userId into constants , so use predefined uuid for dummy user

Reply:
I believe it will be different in at least different artefacts, because it's random, what do you think about usage consortium UUID as a UUID of dummy user? possibly this UUI can be used as a username like `consortiumName_consortiumId`

Reply:
In this case lets use as is with random uuids, if we add this constraint that dummy user id equals consortium id - it can add some complexity to understand for other people why we need this connection. I think the most simple option here - leave random uuids

Reply:
fixed